src/blocks/sandbox-image/imports/index.mjs

```diff
@@ -0,0 +1,324 @@
+		altText = imgObj.alt;
+	} else if ( altSource === 'caption' ) {
+		altText = imgObj.caption;
+	} else if ( altSource === 'title' ) {
+		altText = imgObj.title;
+	} else if ( altSource === 'description' ) {
+		altText = imgObj.description;
+	} else {
+		altText = altSource;
+	}
+
+	/**
+	 * Build the image sources, if defined.
+	 *
+	 * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/source#sizes
+	 */
+	if ( debug ) {
+		// eslint-disable-next-line no-console
+		console.log( 'sourceSizes: ', sourceSizes );
+	}
+	const sourcesTag = []; // tags for picture
+	let srcsetAttribute = ''; // attribute for img
+	const srcsetAttributeArray = []; // holder for srcsetAttribute before using join
+	//
+	sourceSizes.forEach( function ( imgSource ) {
+		const imgSourceObj = getImageData( mediaObj, imgSource.srcset );
+		if ( debug ) {
+			// eslint-disable-next-line no-console
+			console.log( 'imgSource: ', imgSource );
+			// eslint-disable-next-line no-console
+			console.log( 'imgSourceObj: ', imgSourceObj );
+		}
+		if ( imgSourceObj ) {
+			// for picture
+			sourcesTag.push(
+				<source
+					srcSet={ imgSourceObj.src }
+					media={ imgSource.media }
+					type={ imgObj.mime_type }
+				/>
+			);
+			// for imgs
+			srcsetAttributeArray.push(
+				`${ imgSourceObj.src } ${ imgSource.descriptor }`.trim()
+			);
+		}
+	} );
+	srcsetAttribute = srcsetAttributeArray.join( ', ' );
+	if ( debug ) {
+		// eslint-disable-next-line no-console
+		console.log( 'sourcesTag: ', sourcesTag );
+		// eslint-disable-next-line no-console
+		console.log( 'srcsetAttribute: ', srcsetAttribute );
+	}
+	// }
+
+	/**
+	 * Finally, something interesting to display. We have a mediaObject, let's show it!
+	 */
+	return (
+		<div>
+			{ /* Show InspectorControls if the user is able to replace the image or change the focal point. */ }
+			{ ( canEditImage || canEditFocalPoint ) && (
+				<InspectorControls>
+					<PanelBody title={ __( 'Image Settings' ) }>
+						{ canEditImage && (
+							<>
+								<h2>{ __( 'Replace Image' ) }</h2>
+								<PanelRow>
+									{ canEditImage && (
+										// https://github.com/WordPress/gutenberg/blob/HEAD/packages/block-editor/src/components/media-upload/README.md
+										<>
+											<MediaUploadCheck>
+												<MediaUpload
+													onSelect={ ( media ) =>
+														setMediaId( media.id )
+													}
+													value={ currentMediaId }
+													allowedTypes={ [ 'image' ] }
+													render={ ( { open } ) => (
+														<IconButton
+															className="bu-components-image-media-edit-button"
+															onClick={ open }
+															icon="edit"
+															isLarge
+														>
+															{ __(
+																'Edit Media'
+															) }
+														</IconButton>
+													) }
+												/>
+											</MediaUploadCheck>
+											<Button
+												className="bu-components-image-media-remove-button"
```

Comment:
```suggestion
												className="bu-components-image-media-remove-button"
												isSmall
												isSecondary
												isDestructive
```

This will make the removal button red, smaller, and outlined to indicate it is a destructive action.